src/app/admin/components/user-permission-table/user-permission-table.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="table-container">
+  <table mat-table [dataSource]="dataSource">
+    <ng-container matColumnDef="alias">
+      <th mat-header-cell *matHeaderCellDef>Alias</th>
+      <td mat-cell *matCellDef="let user">{{ user.alias }}</td>
+    </ng-container>
+    <ng-container matColumnDef="id">
```

Comment:
What use is ID to the user? Will that tell the user anything? Or is it just here for now and will be removed later on when more information is available?

Reply:
Not sure if the ID is going to be useful. For now it helps to give an overview over all users and their data I suppose. If this view becomes bloated maybe it can be removed.

Reply:
I guess it might become confusing for the user later on, if you remove some users and therefor lack some of the numbers. But yeah, it can wait until we have more information about each user